### ISSUE TITLE
Customizable Main Screen Layout

### ISSUE THREAD

**Issue by Hepticce** [komikku-app/komikku#607]
### Describe your suggested feature

I suggest consider adding a feature that allows users to rearrange the main screen section according to their preferences. This could be implemented as a customization option within the settings appearance. Users could access a 'Rearrange Layout' option, which would provide a drag-and-drop interface where they can move different sections, such as recent reads, favorites, and recommendations, to their desired positions.

![Screenshot_20241227-105003](https://github.com/user-attachments/assets/9c2ebb32-67d4-4c80-9594-ed1e83bac190)
![Screenshot_20241227-111050](https://github.com/user-attachments/assets/92891a8f-9b9f-4cfb-ac33-d51c57ddfcd8)

### Other details

This flexibility would cater to individual user needs, allowing them to prioritize the sections they use most often. For instance, someone who frequently checks their favorites could place that section first, while another user might prefer to prioritize recommendations. Once the arrangement is complete, users could save their changes, and the app would remember their preferences for future sessions.

This approach not only enhances user satisfaction by offering personalization but also aligns with modern app design practices, where user-centered flexibility is a key feature.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have updated the app to version **[1.12.1](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I will fill out all of the requested information in this form.
